felix/bpf/counters/map.go

```diff
@@ -17,34 +17,30 @@ package counters
 import (
 	"encoding/binary"
 
-	"github.com/sirupsen/logrus"
-
 	"github.com/projectcalico/calico/felix/bpf"
 )
 
 const PolicyMapKeySize = 8
 const PolicyMapValueSize = 8
 
 var MapParameters = bpf.MapParameters{
-	Filename:   "/sys/fs/bpf/tc/globals/cali_counters",
 	Type:       "percpu_array",
 	KeySize:    counterMapKeySize, // __u32
 	ValueSize:  counterMapValueSize * MaxCounterNumber,
 	MaxEntries: 1,
 	Name:       bpf.CountersMapName(),
```

Comment:
Should we make it similar to other maps by specifying the Name, Version and do away with ```bpf.CounterMapName```?

Reply:
yes, in another PR that I am working on. Counters are sort of special, per iface and there is some workarounds that need this, so I am leaving it here for now. Jump maps have the same.